### PR TITLE
feat grids added

### PR DESCRIPTION
**Pull request by sonishreyas**
## Change Logs

Feature grids added

## Screenshots
image

## Preview Link
https://deploy-preview-19--ripple-ui.netlify.app/components/grids/grids.html

### CODE REVIEW

**Review by nyyshaaa — COMMENTED** [review]
When hovering over the move to wish-list section like sliding up and down the main container's bottom section also flashes somewhat upward and also others cards flash too minutely  while sliding over wishlist . All else looks good!

File: css/grids.css [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+.cart-billing {
+	grid-area: g-col-2;
+	border-left: 0.2rem solid var(--cta-color);
+	width: max-content;
+	flex-direction: column;
+	align-items: start;
+}
+
+.cart-billing > * {
+	margin: 0 2rem;
```
Comment (Prabhav-Sharma) [inline css/grids.css:23]: Use global variables for spacing for properties like margin, padding, and gap to maintain consistency across the website

Comment (sonishreyas) [reply css/grids.css:23]: Thanks will make that change once I finish with other assignments

**Comment by sonishreyas** [comment]
> When hovering over the move to wish-list section like sliding up and down the main container's bottom section also flashes somewhat upward and also others cards flash too minutely while sliding over wishlist . All else looks good!

Got the issue had given border 0 while hover. Thanks fixed it.

**Pull request merged**